src/ReadingBuddy.java

```diff
@@ -877,21 +937,30 @@ public static void viewAllSearchPosts(){
         }
     }
 
-
-
     public static void enterSearchPostByBook(){
         int seePosts = -1;
+        boolean endNow = false;
```

Comment:
to flag when user should or should not be offered the option to proceed (endNow = true means not eligible to proceed)